R/draws.R

```diff
@@ -1,10 +1,112 @@
-#' TODO
+#' @title Fit the base imputation model and get parameter estimates
+#'
+#' @description Assuming MAR, `draws` fits the base imputation model to the observed data
+#' according to the given multiple imputation methodology and returns the parameter
+#' estimates from the fit.
 #'
 #' @name draws
-#' @param data TODO
-#' @param data_ice TODO
-#' @param vars TODO
-#' @param method TODO
+#' @param data A `data.frame` containing the data to be used in the model.
+#' Must contain the outcome variable. See details.
+#' @param data_ice A `data.frame` that specifies the information related
+#' to the intercurrent events (ICE) and the imputation strategies. Its structure is described in the details.
+#' @param vars A `vars` object as generated by [set_vars()]. See details.
+#' @param method An object returned by the functions [method_bayes()],
```

Comment:
A `method` object as generated by either x , y or z. This specifies the ...

Reply:
done